attest/ake/src/initiator.rs

```diff
@@ -196,3 +196,41 @@ where
         }
     }
 }
+
+/// AuthPending + UnverifiedReport => Ready + VerificationReport
```

Comment:
Is `Ready` really desirable here? Maybe it is better to introduce some other `State` for this that does not actually expose the cipher?

Reply:
Added a new `Terminated` state.